frontend/public/style/_overrides.scss

```
@@ -313,6 +313,11 @@ kbd {
           display: block;
         }
       }
+
+      // Top level items with no sub-items, highlight when active
+      > a.pf-c-nav__link.pf-m-current {
```

Can you remove the `a` ?

```suggestion
      > .pf-c-nav__link.pf-m-current {
```